### PR TITLE
Initial version, python preview release notes

### PR DESCRIPTION
**Pull request by johanste**

### CODE REVIEW

File: docs/release-notes/preview/summary-preview1-python.md [hunk 1]
```diff
@@ -1,30 +1,73 @@
 # Azure SDK for Python Preview 1 Release Notes 
 
-TODO High level info about the release what goals, what libraries are coming with this release, key new features.
+The Azure SDK team is pleased to make available the July 2019 client library preview release. This represents the first release of the ground-up rewrite of the client libraries to ensure consistency, idiomatic design, and excellent developer experience and productivity. This preview release includes new client libraries for AzureCosmos, Identity, Key Vault (keys and secrets), Event Hubs, and Storage (blob, files and queues).
```
Comment (JonathanGiles) [inline docs/release-notes/preview/summary-preview1-python.md]:
```suggestion
The Azure SDK team is pleased to make available the July 2019 client library preview release. This represents the first release of the ground-up rewrite of the client libraries to ensure consistency, idiomatic design, and excellent developer experience and productivity. This preview release includes new client libraries for Azure Cosmos, Identity, Key Vault (keys and secrets), Event Hubs, and Storage (blob, files and queues).
```

File: docs/release-notes/preview/summary-preview1-python.md [hunk 2]
```diff
@@ -1,30 +1,73 @@
...
+If you have a bug or feature request for one of the libraries, please post an issue to [GitHub](https://github.com/auzre/azure-sdk-for-python/issues).
 
 
 ## Changelog
 Detailed change logs are linked to in the Quick Links below. Here are some critical call outs.
 
-TODO: Replace with the top 5 to 10 Features, Known Issues, or Migration pointers.
+### Identity
+
+Azure Identity simplifies authentication across the Azure Python libraries. It supports token authentication using an Azure Active Directory [service principal](https://docs.microsoft.com/en-us/cli/azure/create-an-azure-service-principal-azure-cli) or [managed identity](https://docs.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/overview). Additional forms of authentication (e.g. user authentication) will be added shortly.
```
Comment (JonathanGiles) [inline docs/release-notes/preview/summary-preview1-python.md]: nit: Bullet pointed list for consistency

File: docs/release-notes/preview/summary-preview1-python.md [hunk 3]
```diff
@@ -1,30 +1,73 @@
...
+
+### Cosmos
+
+* New clients corresponding to each level in the object hierarchy (account, container, database etc.) have been created. You can directly access child clients by name.
+
+### Keyvault
+
+* The `azure-keyvault` packages has been split into `azure-keyvault-keys` and `azure-keyvault-secrets`. We intend to add `azure-keyvault-secrets` shortly.
+
+> Note: The `azure-keyvault-keys` and `azure-keyvault-secrets` packages cannot be installed into the same environment as the existing `azure-keyvault` package.
```
Comment (JonathanGiles) [inline docs/release-notes/preview/summary-preview1-python.md]: Nit: The way github renders this note is less than ideal - it makes it less visible than the standard text.

File: docs/release-notes/preview/summary-preview1-python.md [hunk 4]
```diff
@@ -1,30 +1,73 @@
...
+#### Secrets
+
+* A new `azure.keyvault.secrets.SecretClient` type is introduced. 
+* Added support for `async` service methods using the `azure.keyvault.secrets.aio.SecretClient`.
+
+### Storage
+
+#### Blobs
+
+* Create a new set of clients (`BlobServiceClient`, `ContainerClient`, `BlobClient`) corresponding to the storage account, the container and individual blobs respectively.
```
Comment (JonathanGiles) [inline docs/release-notes/preview/summary-preview1-python.md]: Is 'Create' the correct way to start this sentence? Should it be 'Created' or 'Made available'?

File: docs/release-notes/preview/summary-preview1-python.md [hunk 5]
```diff
@@ -1,30 +1,73 @@
...
 
 ## Quick Links
-TODO: fill out table and update links
-| Service  | Install | Quickstart |  API Reference | Changelog |
+
+| Library  | Install | Quickstart |  API Reference | Changelog |
 | -- | -- | -- | -- | -- |
-| Storage Blobs | TODO [package](https://crates.io/crates/azure_sdk_for_rust) | TODO [Readme.md](github.com) | TODO [Preview Documentation](azure.github.io) | TODO [Changelog.md](github.com) |
\ No newline at end of file
+| azure-identity | [package](https://pypi.org/project/azure-identity/) | [Readme.md](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/identity/azure-identity) | [Preview Documentation](https://azure.github.io/azure-sdk-for-python/ref/azure.identity.html) | [History.md](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/keyvault/azure-keyvault-keys/HISTORY.md) |
```
Comment (JonathanGiles) [inline docs/release-notes/preview/summary-preview1-python.md]: The history link for identity is pointing at keyvault-keys.

File: docs/release-notes/preview/summary-preview1-python.md [hunk 6]
```diff
@@ -1,30 +1,73 @@
...
+
+### Keyvault
+
+* The `azure-keyvault` packages has been split into `azure-keyvault-keys` and `azure-keyvault-secrets`. We intend to add `azure-keyvault-secrets` shortly.
+
+> Note: The `azure-keyvault-keys` and `azure-keyvault-secrets` packages cannot be installed into the same environment as the existing `azure-keyvault` package.
+
+#### Keys
+
+* A new `azure.keyvault.keys.KeyClient` type is introduced.
```
Comment (iscai-msft) [inline docs/release-notes/preview/summary-preview1-python.md]: The new 'azure.keyvault.keys.KeyClient` type is introduced.

File: docs/release-notes/preview/summary-preview1-python.md [hunk 7]
```diff
@@ -1,30 +1,73 @@
...
+> Note: The `azure-keyvault-keys` and `azure-keyvault-secrets` packages cannot be installed into the same environment as the existing `azure-keyvault` package.
+
+#### Keys
+
+* A new `azure.keyvault.keys.KeyClient` type is introduced. 
+* Added support for `async` service methods using the `azure.keyvault.keys.aio.KeyClient`.
+
+#### Secrets
+
+* A new `azure.keyvault.secrets.SecretClient` type is introduced.
```
Comment (iscai-msft) [inline docs/release-notes/preview/summary-preview1-python.md]: The new

File: docs/release-notes/preview/summary-preview1-python.md [hunk 8]
```diff
@@ -1,30 +1,73 @@
...
+
+* A new `azure.keyvault.secrets.SecretClient` type is introduced. 
+* Added support for `async` service methods using the `azure.keyvault.secrets.aio.SecretClient`.
+
+### Storage
+
+#### Blobs
+
+* Create a new set of clients (`BlobServiceClient`, `ContainerClient`, `BlobClient`) corresponding to the storage account, the container and individual blobs respectively.
+* Allow callers to specify the blob type in individual method calls rather than instantiating different client types depending on which blob type they want to interact with.
```
Comment (iscai-msft) [inline docs/release-notes/preview/summary-preview1-python.md]: Callers can now specify the blob type

File: docs/release-notes/preview/summary-preview1-python.md [hunk 9]
```diff
@@ -1,30 +1,73 @@
...
+| azure-storage-file | [package](https://pypi.org/project/azure-storage-file/) | [Readme.md](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/storage/azure-storage-file) | [Preview Documentation](https://azure.github.io/azure-sdk-for-python/ref/azure.storage.file.html) | [History.md](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/storage/azure-storage-file/HISTORY.md) |
+| azure-storage-queue | [package](https://pypi.org/project/azure-storage-queue/) | [Readme.md](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/storage/azure-storage-queue) | [Preview Documentation](https://azure.github.io/azure-sdk-for-python/ref/azure.storage.queue.html) | [History.md](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/storage/azure-storage-queue/HISTORY.md) |
+| azure-cosmos | [package](https://pypi.org/project/azure-cosmos/4.0.0b1/) | [Readme.md](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/cosmos/azure-storage-cosmos) | [Preview Documentation](https://azure.github.io/azure-sdk-for-python/ref/azure.cosmos.html) | [changelog.md](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/cosmos/azure-cosmos/changelog.md) |
```
Comment (iscai-msft) [inline docs/release-notes/preview/summary-preview1-python.md]: Why does azure cosmos have changelog.md and the rest have history.md in the repos?

File: docs/release-notes/preview/summary-preview1-python.md [hunk 10]
```diff
@@ -1,30 +1,73 @@
...
+
+* Create a new set of clients (`BlobServiceClient`, `ContainerClient`, `BlobClient`) corresponding to the storage account, the container and individual blobs respectively.
+* Allow callers to specify the blob type in individual method calls rather than instantiating different client types depending on which blob type they want to interact with.
+
 
 ## Quick Links
-TODO: fill out table and update links
-| Service  | Install | Quickstart |  API Reference | Changelog |
+
+| Library  | Install | Quickstart |  API Reference | Changelog |
```
Comment (kaerm) [inline docs/release-notes/preview/summary-preview1-python.md]: What about samples?

File: docs/release-notes/preview/summary-preview1-python.md [hunk 11]
```diff
@@ -1,30 +1,73 @@
...
+pip install --pre azure-keyvault-keys
+pip install --pre azure-keyvault-secrets
+pip install --pre azure-storage-blob
+pip install --pre azure-storage-file
+pip install --pre azure-storage-queue
+```
 
 ## Feedback
-If you have a bug or feature request for one of the libraries, please post an issue to TODO [GitHub](https://github.com/auzre/azure-sdk-for-python/issues) and add the labels Preview and Bug or Feature
+If you have a bug or feature request for one of the libraries, please post an issue to [GitHub](https://github.com/auzre/azure-sdk-for-python/issues).
```
Comment (kaerm) [inline docs/release-notes/preview/summary-preview1-python.md]: 404 - link should be: https://github.com/azure/azure-sdk-for-python/issues

File: docs/release-notes/preview/summary-preview1-python.md (hunk 4 above)
Comment (johanste) [reply docs/release-notes/preview/summary-preview1-python.md]: Fixed wording.

File: docs/release-notes/preview/summary-preview1-python.md (hunk 5 above)
Comment (johanste) [reply docs/release-notes/preview/summary-preview1-python.md]: Thanks - fixed...

File: docs/release-notes/preview/summary-preview1-python.md (hunk 3 above)
Comment (johanste) [reply docs/release-notes/preview/summary-preview1-python.md]: Made it *cursive*

File: docs/release-notes/preview/summary-preview1-python.md (hunk 9 above)
Comment (johanste) [reply docs/release-notes/preview/summary-preview1-python.md]: Good catch! Historical reasons. We'll normalize for beta2.

File: docs/release-notes/preview/summary-preview1-python.md (hunk 11 above)
Comment (johanste) [reply docs/release-notes/preview/summary-preview1-python.md]: Ick.Fixed.

File: docs/release-notes/preview/summary-preview1-python.md [hunk 12]
```diff
@@ -1,30 +1,73 @@
...
-To install all the of the packages, copy and paste the below into a terminal.
+To install the latest preview version of all the of the packages, copy and paste the below into a terminal.
 
-TODO Update code below
-    
-    pip install_command XXX_cosmosFQ XXX_eventhubsFQ XXX_keyvaultFQ XXX_storageFQ
+    `pip install --pre azure-cosmos azure-storage azure-eventhubs azure-keyvault-keys azure-keyvault-secrets azure-storage-blob azure-storage-file azure-storage-queue`
 
-To install them individually
+To install the latest preview version individually:
```
[the remark this answers is not in the document]
Comment (johanste) [reply docs/release-notes/preview/summary-preview1-python.md]: Fixed...

File: docs/release-notes/preview/summary-preview1-python.md (hunk 8 above)
Comment (johanste) [reply docs/release-notes/preview/summary-preview1-python.md]: Changed the wording for this block completely.

File: docs/release-notes/preview/summary-preview1-python.md (hunk 10 above)
Comment (johanste) [reply docs/release-notes/preview/summary-preview1-python.md]: Usage samples are in the readme. Specific samples are linked to from the readme. I don't think that I'd like for people to bounce directly from the announce to a specific sample.

File: docs/release-notes/preview/summary-preview1-python.md [hunk 13]
```diff
@@ -1,30 +1,96 @@
...
+| azure-storage-queue | [package](https://pypi.org/project/azure-storage-queue/12.0.0b1) | [Readme.md](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/storage/azure-storage-queue) | [Preview Documentation](https://azure.github.io/azure-sdk-for-python/ref/azure.storage.queue.html) | [History.md](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/storage/azure-storage-queue/HISTORY.md) |
+| azure-cosmos | [package](https://pypi.org/project/azure-cosmos/4.0.0b1/) | [Readme.md](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/cosmos/azure-cosmos) | [Preview Documentation](https://azure.github.io/azure-sdk-for-python/ref/azure.cosmos.html) | [changelog.md](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/cosmos/azure-cosmos/changelog.md) |
```
Comment (kaerm) [inline docs/release-notes/preview/summary-preview1-python.md]: Is the cosmos package changing now? the link doesn't work

File: docs/release-notes/preview/summary-preview1-python.md [hunk 14]
```diff
@@ -1,30 +1,96 @@
...
+
+* The new `azure-identity` package simplifies authentication across the Azure Python libraries. It supports token authentication using an Azure Active Directory [service principal](https://docs.microsoft.com/en-us/cli/azure/create-an-azure-service-principal-azure-cli) or [managed identity](https://docs.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/overview). Additional forms of authentication (e.g. user authentication) will be added shortly.
+
+### Cosmos
+
+* New clients corresponding to each level in the object hierarchy (account, container, database etc.) have been created. You can directly access child clients by name.
+
+### Keyvault
+
+* The `azure-keyvault` package has been split into `azure-keyvault-keys` and `azure-keyvault-secrets`. We intend to add `azure-keyvault-secrets` shortly.
```
Comment (lmazuel) [inline docs/release-notes/preview/summary-preview1-python.md]: typo, twice secrets makes the sentence incorrect

**Pull request merged**